modules/apps/oauth2-provider/oauth2-provider-scope-impl/src/main/java/com/liferay/oauth2/provider/scope/internal/graphql/OAuth2GraphQLRequestContextValidator.java

```diff
@@ -106,7 +106,7 @@ private void _checkScope(
 
 		if (scopeSheckerType == null) {
 			scopeSheckerType = _getProperty(
-				"http.method", "oauth2.scope.checkertype",
+				"http.method", "oauth2.scope.checker.type",
```

Comment:
In this case, you are right, there is a typo, but I meant `oauth2.scopechecker.type` as I see, it is possible to have the property set in any of the following property names:

- `oauth2.scope.checker.type`
- `oauth2.scopechecker.type`

The first one is tried to be read in the line 104 but if it is null, then the other property is tried to be read.

https://github.com/liferay-appsec/liferay-portal/blob/a313b127aa73822e5856ca3a12ec5d391a62a173/modules/apps/oauth2-provider/oauth2-provider-rest/src/main/java/com/liferay/oauth2/provider/rest/internal/jaxrs/feature/HttpMethodFeature.java#L61

Reply:
You are right! I think this was an original mistake that we then ended up supporting :/

Reply:
Ok! I guess then we will keep supporting the mistake :sweat_smile: